docs/explanations/Cloud-Images-Overview.md

```diff
@@ -0,0 +1,85 @@
+# Ubuntu Cloud Images
+
+Canonical produces new Ubuntu produces new images regularly for various clouds. The Public Cloud team produces, registers, and publishes these images to the Amazon AWS EC2, Amazon AWS EKS, Google GCE, and Microsoft Azure. Other clouds and products, CPC works directly with partners to create a regular release process. These partners include: IBM VPC, IBM Classic, Oracle Cloud Infrastructure (OCI), Google GKE products, and others. CPC also owns the publication of "Premium" images to cloud marketplaces. This includes products such as Ubuntu Pro, Ubuntu Pro Fips, Anbox Cloud, and more. This document will help users understand the different places you can find official Canonical supported Ubuntu images, some of the basic terminology used throughout our docs, and the high level release plan.
+
+## Terminology
```

Comment:
would be cool to have a glossary which explains all of that. see https://squidfunk.github.io/mkdocs-material/reference/tooltips/#adding-a-glossary

Reply:
https://github.com/canonical/ubuntu-cloud-docs/pull/10 is a draft for the glossary like feature.